src/app/app.module.ts

```diff
@@ -1,20 +1,17 @@
-import {NgModule} from '@angular/core';
-
-import {AppRoutingModule} from './app-routing.module';
-import {AppComponent} from './app.component';
-import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
-import {MatSelectCountryModule} from '@angular-material-extensions/select-country';
-import {HttpClientModule} from '@angular/common/http';
-import {MarkdownModule} from 'ngx-markdown';
-import {FlexLayoutModule} from '@angular/flex-layout';
-import {BrowserModule} from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
```

Comment:
here is just your prettier is being differently configured ...

Reply:
It is recommended to have Prettier config files on the project so all contributors use the same while working on it.
Would be a good chance to include it accoding to the [official docs](https://prettier.io/docs/en/configuration.html), what do you think?